src/js/editor/index.js

```diff
@@ -0,0 +1,15 @@
+import toolbox from './toolbox/index';
+import singleEvents from './single_item/index';
+import * as todoEvents from './todo_events';
+
+const init = (itemId) => {
+  todoEvents.getAll().then((items) => {
+    toolbox(items, itemId);
```

Comment:
czytając kod nie wiem co robi funkcja `toolbox`

Reply:
tutaj w tym miejscu przekazuje z pliku js/toolbox.index liste notatek.

Reply:
nie oczekuję Twoich wyjaśnień. 
oczekuję, że kod sam mi powie co robi ;)

przekazujesz w jakim celu? oddaj to w nazwie funkcji.